src/views/Login.tsx

```diff
@@ -15,23 +25,127 @@ export const Login = () => {
     };
     if (!attrs.username || !attrs.password) return;
 
-    console.log({
-      variables: attrs,
-    });
+    try {
```

Comment:
Der er ingen grund til at lave en try catch her, da et graphql request som oftest retunere 200. I stedet skal du tjekke på retur objektet.

ved useMutation

```
const [mutation, {data, error}] = useMutation()
```

paa data og error kan du se hvad du har brug for

Reply:
Jeg tror ikke at jeg ved hvordan jeg skal bruge dem

Reply:
Jeg ved ikke hvordan eller jeg skal tjekke om password og username passer sammen

Reply:
Det fnder backenden ud af. Du kan ogsaa lade det veare as is, ogsaa snakker vi om det imorgen